.travis.yml

```diff
@@ -52,8 +52,9 @@ script:
   - |
       if [[ $TESTS == 'blocks-examples' ]]; then
         cd $TRAVIS_BUILD_DIR
-        git clone https://github.com/mila-udem/blocks-examples.git
+        git clone https://github.com/laurent-dinh/blocks-examples.git
```

Comment:
Hold on, why should I change the PR in `blocks-examples`? I thought I should just merge it...